src/pages/docs/creators/javascript/tutorials/using-reputation.md

```diff
@@ -78,6 +78,32 @@ reputation.setProposalWeights({
 
 In the example provided, the uptime metric is given a higher weight (0.7) than the success rate (0.3). This means that the system will favor providers with better uptime. These weights are flexible and can be adjusted according to the specific needs of your project. By experimenting with these weights, you can find the optimal balance and select providers that best meet your project's requirements.
 
+### Maximizing Performance with the Agreement Selector
+
+The agreement selector is an additional filter that helps you choose the best providers for your tasks. It works in tandem with our proposal filter, to achieve optimal performance.
+
+- **Targeted Filtering:** Our `ProposalFilter` acts as the initial screening tool. It closely examines all available providers, evaluating them against the weights you've defined. Providers that don't meet your weights are eliminated, leaving behind a list of qualified candidates.
+
+- **Precision Selection:** The `agreementSelector` takes over from there. It focuses on the list of providers vetted by our `proposalFilter`. By closely analyzing their performance data, which includes historical single- and multithread CPU benchmark scores, the agreement selector identifies the absolute best fit for your task.
```

Comment:
"By closely analyzing their performance data" - this is a bit much.

It sorts the providers by the weighted scores and chooses one of the highest scoring ones.